app/models/active_guard.rb

```diff
@@ -1,6 +1,6 @@
 class ActiveGuard < Clearance::SignInGuard
   def call
-    if current_user.deleted?
+    if current_user.present? && current_user.deleted?
```

Comment:
You could use `signed_in?` which is part of the base class convenience methods: https://github.com/thoughtbot/clearance/blob/master/lib/clearance/sign_in_guard.rb#L22-L31